devel-tools/cirrus-status/lambda_function.py

```diff
@@ -0,0 +1,83 @@
+import calendar
+import logging
+import os
+from datetime import datetime, timezone
+
+import requests
+from gql import Client, gql
+from gql.transport.aiohttp import AIOHTTPTransport
+
+# Configure logging
+logger = logging.getLogger()
+logger.setLevel("INFO")
+
+BALANCE_LIMIT = 2500.0
+
+# Configuration from environment variables
+GRAPHQL_ENDPOINT = os.getenv("GRAPHQL_ENDPOINT")
+GRAPHQL_TOKEN = os.getenv("GRAPHQL_TOKEN")
+SLACK_WEBHOOK_URL = os.getenv("SLACK_WEBHOOK_URL")
+
+
+def lambda_handler(event, context):
+    graphql_headers = {"Authorization": f"Bearer {GRAPHQL_TOKEN}"}
+    gql_transport = AIOHTTPTransport(url=GRAPHQL_ENDPOINT, headers=graphql_headers)
+    gql_client = Client(transport=gql_transport)
+
+    query = gql(
+        """
+        query GetCreditBalance {
+        ownerInfoByName(platform: "github", name: "zeek") {
+            balanceInCredits
+        }
+        }
+        """
+    )
+
+    message = ""
+
+    try:
+        result = gql_client.execute(query)
+    except gql.TransportQueryError as tqe:
```

Comment:
`gql` resolves to a [function](https://gql.readthedocs.io/en/stable/modules/gql.html#gql.gql) for me, I think we need to instead

```python
from gql.transport import exceptions
```

and then here use `exceptions.TransportQueryError`.

Similarly for all the other exceptions below.